Database/SoulshapeDatabase.lua

```diff
@@ -99,7 +99,7 @@ end
 local function FactionFormatter(faction)
     local name
     if faction.id then
-        name, _ = GetFactionInfoByID(faction.id)
+        name, _ = C_Reputation.GetFactionDataByID(faction.id)
```

Comment:
`C_Reputation.GetFactionDataByID` is (now) returing a table (https://warcraft.wiki.gg/wiki/API_C_Reputation.GetFactionDataByID). I also believe it can be `nil` (if I understand the wiki right).
```suggestion
        name, _ = C_Reputation.GetFactionDataByID(faction.id).name
```

Reply:
fixed pull request